02.async-await/AsyncAwait.Task2.CodeReviewChallenge/Models/Support/IAssistant.cs

```diff
@@ -0,0 +1,9 @@
+﻿using System.Threading.Tasks;
+
+namespace AsyncAwait.Task2.CodeReviewChallenge.Models.Support
+{
+    public interface IAssistant
```

Comment:
Such services should not be placed in `Models` folder. By conversion, folder is used to place `ViewModels` - classes which contains data necessary for models